docs/tools/asf_tools_api.md

```diff
@@ -1,3 +1,3 @@
-# `asf_tools` API Reference
+# `asf_tools` v0.2.0 API Reference
```

Comment:
Should we code-format the version number?
```suggestion
# `asf_tools v0.2.0` API Reference
```

Reply:
this means a change to the `sed` commands, however

Reply:
![image](https://user-images.githubusercontent.com/7882693/124809949-2e57b180-df0d-11eb-87aa-96a840035c5a.png)
vs
![image](https://user-images.githubusercontent.com/7882693/124809990-3f082780-df0d-11eb-9a06-d1342c4051de.png)

I lean towards not code-formatting the version number but I don't feel strongly about it

Reply:
can we bold the "API Reference"

I think I'd prefer all 3 to look slightly different

Reply:
Hmm, this:
![image](https://user-images.githubusercontent.com/7882693/124810597-fdc44780-df0d-11eb-8102-28da49067667.png)
or
![image](https://user-images.githubusercontent.com/7882693/124810657-103e8100-df0e-11eb-8636-14517d572cec.png)

Reply:
Bottom one it is!